bin/run_camera_playlist.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+# This file is part of ts_IntegrationTests.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import asyncio
+
+from lsst.ts.IntegrationTests import RunCameraPlaylist
+from lsst.ts.IntegrationTests.configs.camera_playlist_configs import (
+    atcamera_playlists,
+    cccamera_playlists,
+)
+
+cameras = ["at", "cc"]
+playlists = list(set(list(cccamera_playlists.keys()) + list(atcamera_playlists.keys())))
```

Comment:
Lines 33 to 40 are repeated here and the test. I think you could do this once in the config module file and just use the results in both places. This will change the imports above too.

Reply:
Great idea.  I also moved the list of cameras to the config file.